### PR TITLE
redeem works again

### PR DESCRIPTION
**Pull request by jgfidelis**
erase a bunch of useless manifest dependencies
#### Types of changes

* [x] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/RedeemContent.js [hunk 1]
```diff
@@ -1,32 +1,49 @@
...
-        <h2 className="t-heading-2 mt0 mb4">
-          <FormattedMessage id="address-locator.address-redeem-recurring" />
-        </h2>
-        <div className="vtex-address-modal__redeem">
+  return (
+    <Fragment>
+      <h2 className="t-heading-2 mt0 mb4">
+        <FormattedMessage id="address-locator.address-redeem-recurring" />
+      </h2>
+      <div className="vtex-address-modal__redeem">
```
Comment (lbebber) [inline react/components/RedeemContent.js]: css modules?

Comment (jgfidelis) [reply react/components/RedeemContent.js]: I will do this in another PR!

File: react/components/PickupContent.js [hunk 2]
```diff
@@ -41,24 +34,11 @@ class PickupContent extends Component {
...
-        handleOpenModal={this.handleOpenModal}
-        name={path(['friendlyName'], pickupPoint)}
-        street={address.street}
-        number={address.number}
-      />
-    )
-  }
-
   render() {
+    console.log('teste PickupContent:!!!')
```
Comment (lbebber) [inline react/components/PickupContent.js]: :eyes:

File: react/components/PickupModalContainer.js [hunk 3]
```diff
@@ -91,17 +94,17 @@ class PickupModalContainer extends Component {
   }
 
   render() {
-    const { isModalOpen, storePreferencesData, logisticsQuery, askForGeolocation } = this.props
-    if (!isModalOpen || logisticsQuery.loading) return null
+    const { hasGeolocationPermission } = this.state
+    const { storePreferencesData, logisticsQuery } = this.props
+    if (logisticsQuery.loading || hasGeolocationPermission === undefined) return null
```
Comment (lbebber) [inline react/components/PickupModalContainer.js]: `hasGeolocationPermission === undefined`

tu quer realmente uma comparaçao estrita com `undefined`?

File: react/utils/getCurrentPositionPromise.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+const GEOLOCATION_TIMEOUT = 30 * 1000
+const MAXIMUM_AGE = 3 * 1000
+
+const getCurrentPositionPromise = () => {
```
Comment (lbebber) [inline react/utils/getCurrentPositionPromise.js]: nao acho que precise do `Promise` no nome da funçao, mas ok

Comment (lbebber) [reply react/utils/getCurrentPositionPromise.js]: ah, suponho q seja pra diferenciar da funçao nativa? enfim, fica a teu criterio 🤷‍♂️

File: yarn.lock [hunk 5]
```diff
@@ -0,0 +1,93 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (luciannojunior) [inline yarn.lock]: Isn't this root `yarn.lock` with unnecessary packages?

File: react/components/PickupModalContainer.js (hunk 3 above)
Comment (jgfidelis) [reply react/components/PickupModalContainer.js]: yes, I want it

File: react/utils/getCurrentPositionPromise.js (hunk 4 above)
Comment (jgfidelis) [reply react/utils/getCurrentPositionPromise.js]: sim, é para isso hehe

File: yarn.lock (hunk 5 above)
Comment (jgfidelis) [reply yarn.lock]: I just added int-equalizer, the other came as dependencies

**Pull request merged**